Retain .zst files instead of .gz file extension

isaac-briandt opened the pull request:
Modified the code to successfully check for .zst files and fix failed tests after the modification.

isaac-briandt commented:
@nordzilla kindly review my second bug. Constructive feedbacks are welcome. Thank you!

nordzilla reviewed (changes requested):
> Praise

This looks like a good start. 

Please update the logic regarding the `.zst` file filter, and then you will have to update the test cases that break as a result.

isaac-briandt commented:
Hello sir @nordzilla. Your feedback helped me understand the task better. I implemented the code to retain files with .zst extension only. I had failed tests related to the MIME type as well so I fixed that too. I had to also rename couple of the files to make sure they're in .zst extension

isaac-briandt commented:
And again, to the part that caused me a bit of trouble. I wanted to ask, since It's unfortunate that a widely used Python CLI library would not have uniform error outputs across platforms. Is there anything I could do so that no one else would have to face the issue i faced?

nordzilla commented:
> And again, to the part that caused me a bit of trouble. I wanted to ask, since It's unfortunate that a widely used Python CLI library would not have uniform error outputs across platforms. Is there anything I could do so that no one else would have to face the issue i faced?

That's a great question and a helpful mentality. 

If you would like to open a separate pull request that fixes the test cases on your system (without all of these `.zst` changes), I will review it and merge the code.

isaac-briandt commented:
Thank you for the feedback sir! @nordzilla. I have opened a separate pull request that fixes the test cases
